_dashboards/maps.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Maps
+nav_order: 15
+---
+
+# Maps
+
+With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different data source either directly or using simple or ML-powered aggregations. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
+
+## Getting started
+
+To get started, perform the following steps:
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Maps**.
```

Comment:
It now changed the place to under Plugins. https://github.com/opensearch-project/dashboards-maps/pull/196